pkgs/tools/misc/kronometer/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  kdeDerivation, kdeWrapper, fetchurl, lib,
+  ecm, kdoctools,
+  kconfig, kinit
+}:
+
+let
+  unwrapped = let
+    pname = "kronometer";
+    version = "2.1.0";
+  in kdeDerivation rec {
+    name = "${pname}-${version}";
```

Comment:
is `pname` actually a parameter of `mkDerivation`? I thought it was nothing more than a commonly used name.